src/Microsoft.FeatureManagement/FeatureFilters/TimeWindowFilterSettings.cs

```diff
@@ -14,12 +17,19 @@ public class TimeWindowFilterSettings
         /// An optional start time used to determine when a feature configured to use the <see cref="TimeWindowFilter"/> feature filter should be enabled.
         /// If no start time is specified the time window is considered to have already started.
         /// </summary>
-        public DateTimeOffset? Start { get; set; } // E.g. "Wed, 01 May 2019 22:59:30 GMT"
+        public DateTimeOffset? Start { get; set; } // E.g. "Wed, 01 May 2019 22:59:30 GMT+0800"
 
         /// <summary>
         /// An optional end time used to determine when a feature configured to use the <see cref="TimeWindowFilter"/> feature filter should be enabled.
         /// If no end time is specified the time window is considered to never end.
         /// </summary>
         public DateTimeOffset? End { get; set; } // E.g. "Wed, 01 May 2019 23:00:00 GMT"
+
+        /// <summary>
+        /// An optional list which specifies the recurring time windows used to determine when a feature configured to use the <see cref="TimeWindowFilter"/> feature filter should be enabled.
+        /// The recurring time windows are represented in the form of Crontab expression.
+        /// If any recurring time window filter is specified, to enable the feature flag, the current time also needs to be within at least one of the recurring time windows.
+        /// </summary>
+        public List<CrontabExpression> Filters { get; set; } = new List<CrontabExpression>(); // E.g. ["* 18-19 * * Mon"] which means the recurring time window of 18:00~20:00 on Monday 
```

Comment:
> E.g. ["* 18-19 * * Mon"] which means the recurring time window of 18:00~20:00 on Monday

This looks more like it should belong in a doc than as a comment here. Also this seems kind of like a stray comment, was it intended to be part of the method summary?

Reply:
Ah I see, because we have comments like this on the other properties. Those seem odd. I would suggest the opposite, that we remove the stray comments from the existing properties.

Reply:
I have removed these comments.